enhancements/update/eus-upgrades-mvp.md

```diff
@@ -122,19 +122,24 @@ version away and *block* upgrades for removals which are imminent.
 The MCO, will set Upgradeable=False whenever any MachineConfigPool has one more
 more nodes present which fall outside of a defined list of constraints. For
 instance, if OpenShift has a defined Kubelet Version Skew of N-1, the node
-constraints enforced by the MCO defined in OCP 4.7 could be as follows:
+constraints enforced by the MCO defined in OCP 4.7 (Kube 1.20) would be as follows:
 
 ```yaml
 node.status.nodeInfo.kubeletVersion:
 - v1.20
-- v1.19
 ```
 
-If the policy were to change to allow for a version skew of N-2, v1.18 would be
+If the policy were to change to allow for a version skew of N-2, v1.19 would be
 added to the list of acceptable matches. The MCO is not responsible for defining
 these constrains and constraints are only widened whenever we have CI testing
```

Comment:
While we're in here...

```suggestion
these constraints and constraints are only widened whenever we have CI testing
```

Reply:
Thanks! I gave this section an edit after reading over it again and I've addressed both of those.